### ISSUE TITLE
Passing an array of globs fails

### ISSUE THREAD

**Issue by whydoidoit** [mixu/wildglob#4]
It appears to be expecting the expression parameter to be a string even though the documentation suggests that it can be an array of expressions

**Comment by ryardley** [comment]
Yep having the same problem I tried substituting `multimatch` within the `opts.match` var and this did not work. Be great to see this fixed before having to run searches multiple times or use another lib.
